docs/examples/general/data_loading/dataloading_webdataset.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p>You actually convert from HWC to CHW here (2, 0, 1) and select the one and only channel (img_hwc[0]) - if the image was realy hwc, you would select the first row - and obviously we can see that you select a color plane.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>